src/routes/lobby/LobbyView.vue

```diff
@@ -109,8 +111,8 @@ export default {
   components: {
     LobbyPlayerIndicator,
     BaseSnackbar,
-    TheLanguageSelector,
-  },
+    TheLanguageSelector
```

Comment:
We generally use trailing commas in objects
```suggestion
    TheLanguageSelector,
```

Reply:
Got it